addons/binding/org.openhab.binding.freebox/ESH-INF/thing/thing-types.xml

```diff
@@ -34,6 +34,15 @@
 			<channel id="bytes_down" typeId="bytes_down" />
 		</channels>
 
+		<properties>
+			<property name="vendor">Freebox SAS</property>
+			<property name="serialNumber"></property>
```

Comment:
You mean I should only declare properties for which I already know the value ?
The others would just be added dynamically at runtime ?